### PR TITLE
Add ansible-lint to test pipeline

### PR DESCRIPTION
**Pull request by SuperQ**
* Setup new ansible-lint config with some exceptions
* Fix a number of linter issues.
* Switch to using circleci/python:buster image.

### CODE REVIEW

File: ansible/playbooks/roles/encoder-master/tasks/nfs.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     create: true
     dest: /etc/exports
     block: |
-      /var/lib/sreview	10.0.0.0/8(rw,sync)
+      /var/lib/sreview	10.0.0.0/8(rw,sync)  # noqa 203
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-master/tasks/nfs.yml]: Hrm, now that I think about it, does `/etc/exports` require a hard tab here?

Comment (RichiH) [reply ansible/playbooks/roles/encoder-master/tasks/nfs.yml]: I always to tabs in it

Comment (RichiH) [reply ansible/playbooks/roles/encoder-master/tasks/nfs.yml]: do*

**Comment by SuperQ** [comment]
@RichiH Ok, added some additional cleanup. PTAL.

**Pull request merged**